src/frontend/components/DashboardVideoLive/index.tsx

```diff
@@ -3,7 +3,6 @@ import React, { Fragment, useEffect, useState } from 'react';
 
```

Comment:
I think for additionalContent props, you should order components this way : 
```
              <Fragment>
                <Box direction={'row'} justify={'center'} margin={'small'}>
                  {appData.flags?.live_raw &&
                    video.live_state &&
                    [liveState.IDLE, liveState.PAUSED].includes(
                      video.live_state,
                    ) && <TeacherLiveTypeSwitch video={video} />}
                </Box>
                <DashboardVideoLiveControlPane video={video} />
              </Fragment>
```
Indeed, when <TeacherLiveTypeSwitch /> is not displayed, there is an inelegant white bar on the LiveLayout : 

![image](https://user-images.githubusercontent.com/83247226/164714967-65ed925a-c8fc-407e-80f9-c2459541cfd1.png)

Reply:
I'm not sure to understand what you mean. The code is already the one you are suggesting.